shared/pkg/utils/vulnerability/vulnerability.go

```diff
@@ -45,3 +48,138 @@ func GetSeverityIntFromString(severity string) int {
 		return 0
 	}
 }
+
+func GetCVSSV3VersionFromVector(cvssV3Vector string) string {
+	// example input "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:H"
+	// expected result "3.0"
+	return strings.TrimPrefix(strings.Split(cvssV3Vector, "/")[0], "CVSS:")
+}
+
+// nolint:gomnd
+var v3ScoreValues = map[string]map[string]float64{
+	"AV": {
+		"N": 0.85,
+		"A": 0.62,
+		"L": 0.55,
+		"P": 0.2,
+	},
+	"AC": {
+		"L": 0.77,
+		"H": 0.44,
+	},
+	"PR": {
+		"N": 0.85,
+		"L": 0.62,
+		"H": 0.27,
+	},
+	// Modified scope
+	"PR-MS": {
+		"N": 0.85,
+		"L": 0.68,
+		"H": 0.5,
+	},
+	"UI": {
+		"N": 0.85,
+		"R": 0.62,
+	},
+	"C": {
+		"H": 0.56,
+		"L": 0.22,
+		"N": 0.0,
+	},
+	"I": {
+		"H": 0.56,
+		"L": 0.22,
+		"N": 0.0,
+	},
+	"A": {
+		"H": 0.56,
+		"L": 0.22,
+		"N": 0.0,
+	},
+}
+
+// https://www.first.org/cvss/v3.1/specification-document#7-1-Base-Metrics-Equations
```

Comment:
The calculation of 3.1 and 3.0 is the same?

Reply:
it is, I've added a comment for that to the 3.0 docs